readme.md

```diff
@@ -1303,7 +1303,7 @@ const got = require('got');
 
 ### User Agent
 
-It's a good idea to set the `'user-agent'` header so the provider can more easily see how their resource is used. By default, it's the URL to this repo. You can omit this header by setting it to `null`.
+It's a good idea to set the `'user-agent'` header so the provider can more easily see how their resource is used. By default, it's the URL to this repo. You can omit this header by setting it to `undefined` (or `null`).
```

Comment:
Let's remove support for setting it as `null`. Also update the PR description so we don't forget to mention it in the release notes.

Reply:
Should we throw an error if the header is null?

Reply:
done